sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcOptions.scala

```diff
@@ -27,7 +27,7 @@ import org.apache.spark.sql.internal.SQLConf
 /**
  * Options for the ORC data source.
  */
-private[orc] class OrcOptions(
+private[sql] class OrcOptions(
```

Comment:
I believe `private[sql]` can be removed per [SPARK-16964](https://issues.apache.org/jira/browse/SPARK-16964).

Reply:
Thank you, @HyukjinKwon !